commons/src/main/scala/com/lynbrookrobotics/potassium/commons/electronics/DynamicBraking.scala

```diff
@@ -0,0 +1,41 @@
+package com.lynbrookrobotics.potassium.commons.electronics
+
+import squants.electro.{ElectricPotential, Volts}
+import squants.motion.AngularVelocity
+
+object DynamicBraking {
+  // | -> short
+  // . -> 0 volts
+  private val brakingPatterns = List(
+    "||||||||||||", // strongest braking
+    ".|||||||||||",
+    ".|||||.|||||",
```

Comment:
Shouldn't there be a ".||||.||||.|" or something like that after this?

Reply:
.||||.||||.| == .|||.|||.|||

Reply:
That doesn't seem very obvious, wouldn't having four |s in a row be different then three?

Reply:
No, I think they are pretty similar. What matters is the ratio of '|' to '.'s because each String represents a duty cycle.

Reply:
Okay, if it is solely based on that can you come up with an algorithm for auto generating these patterns?